packages/engine/src/avatar/functions/avatarFunctions.ts

```diff
@@ -96,8 +97,10 @@ const setupAvatar = (entity: Entity, model: any, avatarURL?: string) => {
     if (object.isBone) object.visible = false
     setAvatarLayer(object)
 
-    if (typeof object.material !== 'undefined') {
-      // object.material = object.material.clone()
+    if (object.material) {
+      // Transparency fix
+      object.material.format = RGBAFormat
+
       materialList.push({
         id: object.uuid,
         material: object.material.clone()
```

Comment:
thanks to @HexaField's excellent catch here, he has exposed a great target for a unit test for `setupAvatar`. it would be great to add a small test to assert that the material is indeed a clone, and not an identical reference.

e.g. `it('should make a new material instance per avatar')`

Reply:
put the skeleton for these tests in - loading an avatar from tests is non trivial, as we have to load it using fs of which i can see there being some roadblocks